client/src/modules/stock/exit/exit.js

```diff
@@ -231,7 +238,14 @@ function StockExitController(
   // remove item
   function removeItem(item) {
     vm.stockForm.removeItem(item.id);
+    if (item.lot && item.lot.uuid) {
```

Comment:
Why do you manually remove the selected lot and then call `refreshSelectedLotsList()`?  Shouldn't you only have to do one?